### PR TITLE
[Merged by Bors] - Lint

### PR DESCRIPTION
**Pull request by gavraz**
## Motivation
Lint:
signing
cmd/hare
monitoring
priorityq

### CODE REVIEW

File: monitoring/memoryupdater.go [hunk 1]
```diff
@@ -17,12 +17,14 @@ func bytesToMBFormmater(x uint64) string {
 	return fmt.Sprintf("%vMB", x/1024/1024)
 }
 
+// MemoryUpdater tracks mem stats
```
Comment (antonlerner) [inline monitoring/memoryupdater.go]: such as...

File: monitoring/memoryupdater.go [hunk 2]
```diff
@@ -64,7 +67,7 @@ func (mu *MemoryUpdater) Update() {
 	mu.recordsCount++
 }
 
-// Status - returns a string description of the current status
+// Status returns a string description of the current status
```
Comment (antonlerner) [inline monitoring/memoryupdater.go]: of what?

File: monitoring/tracker.go [hunk 3]
```diff
@@ -2,13 +2,15 @@ package monitoring
 
 import "math"
 
+// Tracker tracks uint64 values
```
Comment (antonlerner) [inline monitoring/tracker.go]: please elaborate more on these

**Review by antonlerner — APPROVED** [review]

File: monitoring/memoryupdater.go (hunk 1 above)
Comment (gavraz) [reply monitoring/memoryupdater.go]: Fixed

File: monitoring/memoryupdater.go (hunk 2 above)
Comment (gavraz) [reply monitoring/memoryupdater.go]: Fixed

File: monitoring/tracker.go (hunk 3 above)
Comment (gavraz) [reply monitoring/tracker.go]: Fixed

**Comment by gavraz** [comment]
bors merge

**Pull request closed without merging**